modules/sfp_flickr.py

```diff
@@ -144,23 +167,61 @@ def handleEvent(self, event):
 
             self.sf.info("Parsing page " + str(page) + " of " + str(pages))
 
-            # Extract emails
+            # Extract data
             for photo in data['photos']['photo']:
```

Comment:
I'd prefer the use of `.get()` here since there is no guarantee that `photos` or `photo` exist as keys.

Reply:
`photos` is checked a few lines above.